atom/browser/native_browser_view.cc

```diff
@@ -2,9 +2,10 @@
 // Use of this source code is governed by the MIT license that can be
 // found in the LICENSE file.
 
-#include "atom/browser/native_browser_view.h"
+#include <vector>
 
 #include "atom/browser/api/atom_api_web_contents.h"
+#include "atom/browser/native_browser_view.h"
```

Comment:
Suprised linting doesn't complain about this, standard afaik is to have the header file for the current cc file as the first thing in the file (like it was originally).  If you put this back at the top I'll approve it 😀